### DISCUSSION TITLE
urgent!!!!   errors at installation of security onion 2.4.40

### DISCUSSION THREAD

**Discussion by rajeshkandula369** [Security-Onion-Solutions/securityonion#12312] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

12

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

i got a error at last step of installation and I find the errors by running the /root/errors.log
![urgent](https://github.com/Security-Onion-Solutions/securityonion/assets/158308916/f041264b-7fcc-4454-9425-cf231724c659)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by petiepooo** [comment]
Welcome to Github and Security Onion.  I see this is your first post ever here, so a little tip:  I cannot see how an error on an Eval-type installation could ever be urgent, and this is a community-based support forum.  Please don't flag things as urgent when they are not; it will not make people more eager to help, and you will likely be ignored.

With that said, it looks like there was some issue creating the certificate/key pairs during install, but the screenshot you posted does not have the details.  Please search through the /root/sosetup.log file for the first instance of "Result: False", then look at the lines before that that start with "[INFO    ]" for more information on why that state task failed.  You can respond here with that and someone may be able to help.  Text pastes are better than screenshot images.

You could also just try running the setup a second time.

**Reply by rajeshkandula369** [reply]
Apologies for the urgency, but I need to address this promptly as I'm using the Security Onion installation for an assignment, and the deadline for submission has passed. Upon reviewing the `/root/sosetup.log` file, I encountered the following errors.

Enabled schedule on minion.
result:
True
INFO | Verifying setup
2824-02-05T18:32:442 |
WARNING: Errors detected during setup.
ERRORS
Result: False
Comment: One or more requisite failed: ssl.influxdb_crt
Result: False
Result: False
Comment: One or more requisite failed: ssl.redis_crt
Result: False
LERROR 1 Command '/opt/so/_salt.tmp.a2caht5y' failed with return code: 1
CERROR
CERROR
] stdout:
1 retcode: 1
[ERROR 1 ('pid': 388875, 'retcode': 1, 'stdout': '\nInstallation initiated, vi
ew install log for further details.', 'stderr': ''}
[ERROR 1 Command /opt/so/_salt.tmp.zi3eep41' failed with return code: 1
CERROR
1 stdout:
[ERROR 1 retcode: 1
[ERROR] ['pid': 398277, 'retcode': 1, 'stdout': 'AnInstallation initiated, vi
ew install log for further details. 'stderr': }
Result: False
[analyst@localhost ~]$
![sosetup log](https://github.com/Security-Onion-Solutions/securityonion/assets/158308916/109b9d2f-ae54-40b8-95eb-1126d93afe6b)

**Reply by petiepooo** [reply]
That is essentially the same as what you pasted before, the collection of errors at the end of the file.  What I asked for was the _first_ appearance of `Result: False` in sosetup.log and the block of INFO lines above it.  That will have information about what failed.

If it's during the certificate creation (state ssl.init), your best bet is probably just run so-setup again...

Best of luck with your schoolwork.

**Reply by rajeshkandula369** [reply]
Thank you, sir. I've successfully installed Security Onion in Evaluation mode. However, I'm currently facing an issue where I'm unable to access the web console on Chrome. During the setup process, I configured one adapter as bridged and another as NAT. I assigned an IP address identical to that of my physical computer.

**Reply by petiepooo** [reply]
When a packet with that IP arrives, how does your computer know where to send it?  

Two devices on a network should _never_ have the same IP.  Reinstall and use a different IP.  

Your admin adapter can be bridged, but your monitor interface needs a different configuration.  Usually, that's a dedicated interface, even for VMs.  Reread https://docs.securityonion.net/en/2.4/hardware.html#packets

**Reply by rajeshkandula369** [reply]
"Thank you, sir. I have finally completed the installation successfully and am now able to access the web console."

**Reply by foxman0719** [reply]
hola, como lo soluciono? yo tengo el mismo problema

**Reply by petiepooo** [reply]
Estás comentando una discusión cerrada. No esperes una respuesta aquí. 

Lea el contenido del enlace publicado anteriormente.  La máquina virtual debe tener dos interfaces, la primera con una dirección IP a la que se accede a la GUI y la segunda sin una IP que esté conectada a algo que envíe paquetes.

Si aún necesita ayuda, abra una nueva discusión, pero intente usar el inglés.
